python/MDSplus/__init__.py

```diff
@@ -86,18 +86,18 @@ class MDSplusVersionInfo(ctypes.Structure):
                        ("MDSVERSION",ctypes.c_char_p)]
         try:
             _info=ctypes.cast(libs.MdsShr.MDSplusVersion,ctypes.POINTER(MDSplusVersionInfo)).contents
-            _ver = version.tostr(_info.MDSVERSION.decode())
+            _verchk = _ver.tostr(_info.MDSVERSION.decode())
         except:
-            _ver = "Unknown"
-        if _ver != __version__ or _ver == "Unknown":
+            _verchk = "Unknown"
+        if _verchk != __version__ or _verchk == "Unknown":
             sys.stderr.write('''Warning:
   The MDSplus python module version (%s) does not match
   the version of the installed MDSplus libraries (%s).
-  Upgrade the module using the mdsplus/mdsobjects/python directory of the
-  MDSplus installation.
-''' % (__version__, _ver ))
+  Upgrade the module using the mdsplus/python/MDSplus directory of the
+  MDSplus installation or set PYTHONPATH=/usr/local/mdsplus/python/MDSplus.
+''' % (__version__, _verchk ))
     version_check()
-del version_check, _ver
```

Comment:
well the idea was to remove all trace of the version_check for the final module. which is why I del all used vars.. I at some point rename _ver to version to make it more transparent but rebasing or something caused it to scramble it seems.. hte _mimport should write to version and all _ver.* should be version.*

Reply:
I rename the _ver to version